### DISCUSSION TITLE
last seen and online status for non chat contacts

### DISCUSSION THREAD

**Discussion by MarcKete** [rizwansoaib/whatsapp-monitor#138] [General]
1 day ago it seems that the "last seen" and "online" status of people in your address book but have never chatted with, has completely dissapeared. The profile picture still displays (so I assume I'm not blocked). I know the last seen status can be set to "contacts only", but normally the online status should always remain visible if this person comes online. 

I cannot determine whether this is as result of a new WA release, a regulatory limitation, ...  Do you epxerience the same issue as well? Thanks for your feedback

**Comment by rizwansoaib** [comment]
Yes already one issue created

**Reply by MarcKete** [reply]
Hi Rizwan, thanksfor the swift reply. I know I already created an issue on this, but the "phenomena" (nothing show any longer) just appaered 1 day ago. Previously you still could see once that contact appeared online (on your mobile), but now it simply doesn't display anything any longer.

**Reply by rizwansoaib** [reply]
Yes I saw it was working  but since 2 days many paid apps also facing issues. 


But I will check.

**Reply by MarcKete** [reply]
great, always appreciate your effort and devotion! ;-)

**Reply by don199910** [reply]
I have talked to whatsapp support and they told this is a new security feature, Im afraid. Is there a way through this?

**Comment by rizwansoaib** [comment]
Many app store apps which are paid those also facing issues

**Comment by rizwansoaib** [comment]
![Screenshot_2021-12-08-22-41-47-789_com android vending](https://user-images.githubusercontent.com/29729380/145252825-762f55eb-a7f1-4358-9ca9-1ba81ffbeb5f.jpg)
![Screenshot_2021-12-08-22-41-52-726_com android vending](https://user-images.githubusercontent.com/29729380/145252843-2b80bef0-eb8b-4470-9705-85e6027b974a.jpg)



08 Dec 2021 many paid apps are facing issues

**Comment by rizwansoaib** [comment]
Can you send that reply.

On Wed, 8 Dec, 2021, 10:46 pm don199910, ***@***.***> wrote:

> I have talked to whatsapp support and they told this is a new security
> feature, Im afraid. Is there a way through this?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/138#discussioncomment-1774189>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZE4UPTF7FM7MNQGMB3UP6HFPANCNFSM5JTOSIFA>
> .
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Reply by don199910** [reply]
yes [](url)
<img width="391" alt="wa" src="https://user-images.githubusercontent.com/95769344/145257419-0da3dc1c-195c-4869-8b80-faeb3a644e05.PNG">

**Comment by rizwansoaib** [comment]
To me which whatsapp team send you

On Wed, 8 Dec, 2021, 10:58 pm Rizwan Ahmad, ***@***.***>
wrote:

> Can you send that reply.
>
> On Wed, 8 Dec, 2021, 10:46 pm don199910, ***@***.***> wrote:
>
>> I have talked to whatsapp support and they told this is a new security
>> feature, Im afraid. Is there a way through this?
>>
>> —
>> You are receiving this because you commented.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/138#discussioncomment-1774189>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AHC2EZE4UPTF7FM7MNQGMB3UP6HFPANCNFSM5JTOSIFA>
>> .
>>
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Comment by MarcKete** [comment]
Hope a workaround can be found for this.

**Reply by don199910** [reply]
will this be permanent

**Reply by rizwansoaib** [reply]
Paid app developers fix this issues now those apps are working fine.

**Reply by MarcKete** [reply]
so there's still hope?

**Reply by don199910** [reply]
when will this be updated on whatsapp monitor :(

**Comment by rizwansoaib** [comment]
Currently I am not able to figure this out how to fix this.

On Thu, 9 Dec, 2021, 6:22 pm don199910, ***@***.***> wrote:

> when will this be updated on whatsapp monitor :(
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/138#discussioncomment-1778870>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZFS4EMFS3LPZDCP3YTUQCRAZANCNFSM5JTOSIFA>
> .
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Reply by MarcKete** [reply]
Sure there must be a way, if commercial apps can do the same. Prefer to stay here in order to support you

**Reply by MarcKete** [reply]
Seems to be getting quiet here on this topic... anyone any progress or ideas so far?
